src/main/java/Duke.java

```diff
@@ -1,10 +1,22 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo);
+        System.out.println("Hello from Duke!");
+        System.out.println("What can I do for you?");
+
+        String taskInput;
+        Scanner sc = new Scanner(System.in);
```

Comment:
In general, try using English words as variables; it makes it easier to understand your code, especially further down